src/components/OptionsList.js

```diff
@@ -38,6 +37,9 @@ const propTypes = {
     // Whether to show headers above each section or not
     hideSectionHeaders: PropTypes.bool,
 
+    // A flag to indicate wheter the option is on the RightDockedModal or not
+    isOnRightDockedModal: PropTypes.bool,
```

Comment:
It might make more sense to name this for the behavior we want to allow or prevent (which is focusing options with arrow keys) rather than where it will be used.

What if instead this was called `canFocusOptions`?

Reply:
I actually wanted to discuss this variable. 
Right now it is not just controlling the focus, I created it and went hard on it, just want to get some stuff working.
Right now it is controlling:
- Whether the arrow keys should be on or off in `OptionsSelector`;
- Whether the option can be focused in `OptionsList`;
- Both the text style and if should show the Pencil and PinCircle icons in `OptionRow`.

Maybe we could name it:
- `disableArrowKeysActions` when in the `OptionsSelector` prop;
- `canFocusOptions` when in the `OptionsList` prop;
- `showAdditionalOptionStates` (copied from your review below) in the `OptionRow`

Reply:
That plan sounds perfect to me.

Reply:
Decided to take the same approach as changing `showAdditionalOptionState` to `hideAdditionalOptionState`, and changed `canFocusOptions` to `disableFocusOptions`.

Also separated it into 2 props on `<OptionsSelector />`. Because there is no correlation between `disableArrowKeysActions` and `hideAdditionalOptionStates`, both should come from this parent.

Reply:
Hey, does anyone recall why we desire the disabling of the arrow keys for the right docked modal? An issue was opened for enabling the arrow keys, but I can only assume we had a good reason for blocking this in the first place?

Reply:
Sorry, I don't recall why.